src/components/landing/SliderForEvents/SliderForEvents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
```

Comment:
All styles imports should be places after all modules imports. Move it to bottom

Reply:
I moved it to bottom.
DONE